patches/tModLoader/Terraria/Localization/Content/pl-PL/tModLoader.json

```diff
@@ -43,45 +42,45 @@
+		"AskForModIndex": "Wpisz liczbę, aby przełączać między włączonymi/wyłączonymi modami",
+		"AskForCommand": "Wpisz komende: ",
 
-		// Interface
-		// "BossBarStyle": "Boss Bar Style: {0}",
-		// "BossBarStyleNoOptions": "No Options",
+		// Interface // Interfejs
+		"BossBarStyle": "Styl Paska Bossa: {0}",
+		"BossBarStyleNoOptions": "Bez Opcji",
 
-		// Mod Errors
+		// Mod Errors // Błędy Modów
 		"LoadError": "Wystąpił błąd podczas ładowania {0}",
-		// "LoadErrorDisabled": "The mod(s) have been automatically disabled.",
-		// "LoadErrorCulpritUnknown": "The mod responsible is unknown and tModLoader must be restarted.",
+		"LoadErrorDisabled": "Mod(y) zostały automatycznie wyłączone.",
+		"LoadErrorCulpritUnknown": "Mod odpowiedzialny jest nieznany i tModLoader musi zostać ponownie uruchomiony.",
 		"LoadErrorVersionMessage": "Wykryto mod dla tModLoader v{0}\nUżywasz tModLoader v{1}",
-		// "LoadErrorDotNet45Required": ".NET Framework 4.5 or newer is required to use mods. Please download the latest .NET Framework",
+		"LoadErrorDotNet45Required": ".NET Framework 4.5 lub nowszy jest wymagany, aby używać modów. Proszę, pobierz najnowszy .NET Framework",
 		"LoadErrorDependencyMissing": "Brakujący mod: {0} wymagany przez {1}",
 		"LoadErrorErrorReadingModFile": "Błąd podczas ładowania pliku moda {0}",
 		"LoadErrorHashMismatchCorrupted": "Niepoprawny hash, dane moda zostały zmodyfikowane lub uszkodzone",
 		"LoadErrorDependencyVersionTooLow": "{0} wymaga {2} w wersji {1}, ale zainstalowana jest {3}",
-		// "LoadErrorMajorVersionMismatch": "{0} targets {2} version {1} but you have a newer major version ({3}) which may not be compatible. {0} must be updated.",
-		// "LoadWarningBannerOrBannerItemNotSet": "{0}: {1}. You have set ModNPC.banner or ModNPC.bannerItem, but not the other. Make sure you set both, check your spelling, and that the item exists if you are using mod.ItemType(). Report this to the {0} developers.",
-		// "UnloadError": "One or more errors occured while unloading and tModLoader must be restarted to prevent further issues.",
-		// "DefensiveUnload": "{0} was unable to unload properly. Modders must use defensive programming to ensure unloading completes regardless of any errors during loading.",
-		// "BuildError": "An error ocurred while building {0}",
-		// "CompileError": "Compiling {0} failed with {1} errors and {2} warnings",
+		"LoadErrorMajorVersionMismatch": "{0} celuje w {2} wersje {1} ale masz nowszą wersję główną ({3}) które mogą być niezgodne. {0} muszą zostać zaktualizowane.",
+		"LoadWarningBannerOrBannerItemNotSet": "{0}: {1}. Ustawiłeś ModNPC.banner lub ModNPC.bannerItem, ale nie inne. Upewnij się, że ustawiasz oba, sprawdź pisownię i że element istnieje, jeśli używasz mod.ItemType(). Zgłoś to deweloperom {0}.",
```

Comment:
Ustawiłeś ModNPC.banner lub ModNPC.bannerItem, ale nie oba. Upewnij się, że ustawiasz oba, sprawdź pisownię i że element istnieje, jeśli używasz mod.ItemType(). Zgłoś to deweloperom {0}.",